README.md

```diff
@@ -185,8 +185,9 @@ sudo make install
 
 ### Windows (Cygwin)
 
-DABlin can also be compiled on Windows using [Cygwin](https://cygwin.com/). The following
-Cygwin packages are required:
+DABlin can also be compiled on Windows using 
```

Comment:
In this case (and also further cases) it can/will happen that the link URL is too long - sometimes also too long for a single line. Therefore it is fine to just consider the link text when ensuring that the desired line character limit is not exceeded.